server/routes/files.js

```diff
@@ -16,4 +20,21 @@ router.get('/files', function (req, res) {
 });
 
 
+router.get('/media-files', (req, res) => {
+  fileExplorer.scanForMedia(req.query.path, (err, filepaths) => {
+    if (err) {
+      res.send(500).json({message: err.message});
```

Comment:
`return res...` would suffice.

Reply:
Also, it was a bug in my previous code, the correct code will be -
`res.status(500).json({//json});` not `res.send`